client/app/components/packages/land-use-action.hbs

```diff
@@ -25,47 +25,56 @@
   {{/if}}
 
   {{#each this.sortedSelectedActions as |landUseAction|}}
-    <Ui::Question
-      @group={{true}}
-      class="fieldset relative scale-fade-in"
-      as |Q|
-    >
-      <Q.Prompt data-test-action-name="{{landUseAction.name}}">
+    <fieldset class="fieldset relative scale-fade-in">
+      <Ui::Legend data-test-action-name="{{landUseAction.name}}">
         {{landUseAction.name}}
-      </Q.Prompt>
-
-      {{#if (or (eq landUseAction.name "Zoning Special Permit") (eq landUseAction.name "Zoning Certification") (eq landUseAction.name "Zoning Authorization") (eq landUseAction.name "Zoning Map Amendment") (eq landUseAction.name "Zoning Text Amendment") (eq landUseAction.name "Modification") (eq landUseAction.name "Renewal"))}}
-        {{#if (or (eq landUseAction.name "Zoning Special Permit") (eq landUseAction.name "Zoning Certification") (eq landUseAction.name "Zoning Authorization"))}}
-          <div class="grid-x grid-margin-x">
-            <div class="shrink cell">
-              {{!-- TODO:
-                Ui::Question component doesn't work when a grid splits up label and input.
-                The label needs a functioning for="" if it doesn't wrap the input.
-              --}}
-              <label class="middle">
-                <strong>How many {{landUseAction.name}} actions?</strong>
-                <Ui::RequiredAsterisk />
-              </label>
+      </Ui::Legend>
+
+      {{#if (or
```

Comment:
Thank you for the reformatting! 🙏  Looks so much clearer. However this highlights  that these land use names should be moved into an array within the component. like `["Zoning Special Permit", "Zoning Certification"]` 
Then in the template we can use a helper like `{{in-array}}`

Reply:
Let's create a separate issue for this?

Reply:
Sounds good